### PR TITLE
[WIP] fix: respect `call` inside `staticlookup`

### PR DESCRIPTION
**Pull request by evilebottnawi**
partial https://github.com/glayzzle/php-parser/issues/185

### CODE REVIEW

File: src/parser/variable.js [hunk 1]
```diff
@@ -100,6 +100,74 @@ module.exports = {
     return result(what, offset);
   },
 
+  read_what: function(name, is_static_lookup = false) {
```
Comment (ichiriac) [inline src/parser/variable.js]: the name variable is never used

File: src/parser/variable.js [hunk 2]
```diff
@@ -140,98 +208,24 @@ module.exports = {
...
-              this.error([this.tok.T_STRING, this.tok.T_VARIABLE]);
-              // graceful mode : set what as error mode & continue
-              what = this.node("identifier");
-              name = this.text();
-              this.next();
-              what = what(name);
-              break;
-          }
-          result = node(result, what);
+          result = node(result, this.read_what(name));
```
Comment (ichiriac) [inline src/parser/variable.js]: the name paramter is extra, not used

**Comment by ichiriac** [comment]
ok, the read_what function is correct, but you may only avoid name as parameter as it's not used

**Comment by evilebottnawi** [comment]
@ichiriac WIP

**Comment by evilebottnawi** [comment]
done :star:

**Comment by ichiriac** [comment]
its all right

**Pull request merged**